polaris.shopify.com/content/whats-new/version-11-layout.md

```diff
@@ -34,7 +34,7 @@ Creating new layout components and applying them to the existing Polaris compone
 
 ### Removed components
 
-The original Stack and Card components have been removed and replaced with [LegacyStack](https://polaris.shopify.com/components/layout-and-structure/legacy-stack) and [LegacyCard](https://polaris.shopify.com/components/layout-and-structure/legacy-card). We plan to replace LegacyStack and LegacyCard with the new [Card](https://polaris.shopify.com/components/layout-and-structure/card) and [VerticalStack](https://polaris.shopify.com/components/layout-and-structure/vertical-stack) components in the next major release.
+The original Stack and Card components have been removed and replaced with [LegacyStack](https://polaris.shopify.com/components/layout-and-structure/legacy-stack) and [LegacyCard](https://polaris.shopify.com/components/layout-and-structure/legacy-card). We plan to replace LegacyStack and LegacyCard with the new [Card](https://polaris.shopify.com/components/layout-and-structure/card) and [BlockStack](https://polaris.shopify.com/components/layout-and-structure/block-stack) components in the next major release.
```

Comment:
Should we leave references to VerticalStack as is on this page? That way documentation isn't confusing for users who are bumping from <=v10 to v11 and they don't migrate to a component that doesn't yet exist on v11. And then this VerticalStack to BlockStack update can be included in a section for renamed components in a new `whats-new/version-12-abcde` for users who are already on v11 and upgrading to v12

Reply:
I was on the fence about this as well but decided to make the change since it's a public page. Hopefully it won't be around much longer but the links will be confusing anyway. I'm still on the fence. Leave the wording but update the links?

Reply:
Just realized that with the redirects, links to vertical stack with redirect to block stack anyways so I think it's fine to merge what you already have, sorry 😅  

Agree the links will be a bit confusing but since doc changes will be going out with v12 and a migration guide/what's new page for v12, I think comms will help alleviate some of that 🤞